package.json

```diff
@@ -48,8 +48,8 @@
     "build": "npm run rollup && npm run uglify",
     "lint": "eslint .",
     "lintfix": "npm run lint -- --fix",
-    "rollup": "rollup -c build/rollup-config.js",
-    "watch": "rollup -w -c build/rollup-config.js",
+    "rollup": "rollup -c build/rollup-config.mjs",
```

Comment:
Since we now load the config as ESM, we need to use the `.mjs` extension. Once we have moved to a pure ESM package this can be restored.